ksqldb-engine/src/main/java/io/confluent/ksql/security/DefaultKsqlPrincipal.java

```diff
@@ -37,4 +37,8 @@ public String getName() {
   public Map<String, Object> getUserProperties() {
     return Collections.emptyMap();
   }
+
+  public Principal getOriginalPrincipal() {
```

Comment:
Could you add a comment about where this method is used? Just to make sure is never deleted or public id removed accidentally.

Reply:
Done.